azcopy/syncProcessor.go

```diff
@@ -148,80 +146,64 @@ func newInteractiveDeleteProcessor(deleter traverser.ObjectProcessor, deleteDest
 
 const LocalFileObjectType = "local file"
 
-func newSyncLocalDeleteProcessor(cca *cookedSyncCmdArgs, fpo common.FolderPropertyOption) *interactiveDeleteProcessor {
-	localDeleter := localFileDeleter{rootPath: cca.destination.ValueLocal(), fpo: fpo, folderManager: common.NewFolderDeletionManager(context.Background(), fpo, common.AzcopyScanningLogger)}
-	return newInteractiveDeleteProcessor(localDeleter.deleteFile, cca.deleteDestination, LocalFileObjectType, cca.destination, cca.incrementDeletionCount)
+func NewLocalFileDeleter(fpo common.FolderPropertyOption) localFileDeleter {
```

Comment:
ditto

Reply:
will do a pass after migrating copy too to make sure there are no methods that dont need to be exported